__tests__/app.test.js

```diff
@@ -91,3 +91,52 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: should return an array of all the articles with comment_counts", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(13);
```

Comment:
Great to see you're checking the length before you start the loop to prevent any false positives if the array was empty 👍